modular_skyrat/modules/mapping/code/lighting.dm

```diff
@@ -0,0 +1,4 @@
+// A "daylight" light emitter that's "safe" to flood-fill outdoors areas with. I hate this. - Rimi
+/obj/effect/light_emitter/interlink
+	set_luminosity = 4
+	set_cap = 0.5
```

Comment:
Also fiddle with the color here, if its possible.
Interlink exterior should be cozy, not entirely white (if you can't make it look nice don't even bother)

Reply:
Those are outdoors light emitters, that won't effect the interlink, and they also do not support colour as far as I know.

Reply:
Yea, the light colour var isn't used on these.